main.go

```diff
@@ -200,6 +207,14 @@ func stopWithRconCli() error {
 	return rconCliCmd.Run()
 }
 
+func announceStopViaConsole(logger *zap.Logger, stdin io.Writer, shutdownDelay time.Duration) {
+	logger.Info("Sending shutdown announce 'say' to Minecraft server...")
+	_, err := stdin.Write([]byte(fmt.Sprintf("say Server shutting down in %0.f seconds\n", shutdownDelay.Seconds())))
```

Comment:
Floating format might look overly "precise". How about just integer formatting with

```suggestion
	_, err := stdin.Write([]byte(fmt.Sprintf("say Server shutting down in %d seconds\n", shutdownDelay.Seconds())))
```

Reply:
I tried int formatting initially - but the `.Seconds()` method returns a float. The `%0.f` format code specifies 0 decimal precision for floats, so the effect ends up the same as `%d`

* Go playground https://play.golang.org/p/zw-EkgTPoSL

Reply:
Aha, I mentioned that without thinking what the return type was.